packages/web-react/src/components/FileUploader/useFileUploaderStyleProps.ts

```diff
@@ -69,7 +72,9 @@ export const useFileUploaderStyleProps = (props?: FileUploaderStyleProps): FileU
         link: classNames(fileUploaderInputLinkClass, ...fileUploaderInputLinkUtilityClasses),
         validationText: fileUploaderInputValidationTextClass,
         dropZone: {
-          root: fileUploaderInputDropZoneClass,
+          root: classNames(fileUploaderInputDropZoneClass, {
+            [fileUploaderInputDropZoneHiddenClass]: props?.isDropZoneHidden && props.queueLimitBehavior === 'hide', // TODO: Modify when the disabled state is ready for the entire component
```

Comment:
Please add an issue link to this comment.

Reply:
done